web/context.go

```diff
@@ -627,3 +627,14 @@ func (c *Context) RequireBotUserId() *Context {
 	}
 	return c
 }
+
+func (c *Context) RequirePrivacy() *Context {
+	if c.Err != nil {
+		return c
+	}
+
+	if c.Params.Privacy != "public" && c.Params.Privacy != "private" {
```

Comment:
Same here for the constant part

Reply:
Done https://github.com/mattermost/mattermost-server/pull/14287/commits/045457a23d4112cf390b6ae5f244a0ae194f1ec8